examples/helm/ha_cluster_with_proxy_and_password.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#
+# This values file overrides the default kube-starrocks chart to:
```

Comment:
Since this is a document, it is best to place it in the 'doc' directory rather than the 'examples' directory. Our 'examples' directory contains some CRD yaml files, which are used to deploy an SR cluster directly via kubectl.